src/index.js

```diff
@@ -219,17 +225,19 @@ export default async function microbundle(inputOptions) {
 
 	let cache;
 	let out = await series(
-		steps.map(({ inputOptions, outputOptions }) => async () => {
+		steps.map(config => async () => {
+			const { inputOptions, outputOptions } = config;
 			inputOptions.cache = cache;
 			let bundle = await rollup(inputOptions);
 			cache = bundle;
-			const { code } = await bundle.write(outputOptions);
-			return await getSizeInfo(code, outputOptions.file);
+			await bundle.write(outputOptions);
+			return await config._sizeInfo;
+			// return await getSizeInfo(code, outputOptions.file, options.raw);
```

Comment:
You've commented here, why if you don't mind me asking?

Reply:
oh that should be removed